vmm_core/virt_mshv/src/lib.rs

```diff
@@ -1289,7 +1317,11 @@ struct MshvHypercallHandler<'a, T> {
 impl<T: CpuIo> MshvHypercallHandler<'_, T> {
     const DISPATCHER: hv1_hypercall::Dispatcher<Self> = hv1_hypercall::dispatcher!(
         Self,
-        [hv1_hypercall::HvPostMessage, hv1_hypercall::HvSignalEvent],
+        [
+        hv1_hypercall::HvPostMessage, 
+        hv1_hypercall::HvSignalEvent,
+        hv1_hypercall::HvExtQueryCapabilities
```

Comment:
Why is this necessary now?

Reply:
It's not necessary. It was just quietening a warning message I was seeing in the code. I can remove if you think that's a better option.

Reply:
Removed in the latest iteration.